components/src/dynamo/planner/manual/scaling/scaling_e2e.py

```
@@ -58,19 +65,17 @@ def __init__(
     def _run_kubectl(self, cmd: List[str]) -> Tuple[bool, str]:
-        """Run kubectl command and return success status and output."""
         try:
             result = subprocess.run(cmd, capture_output=True, text=True, timeout=30)
             return result.returncode == 0, result.stdout.strip()
         except subprocess.TimeoutExpired:
-            logger.error(f"kubectl command timed out: {' '.join(cmd)}")
+            logger.error("kubectl command timed out: %s", " ".join(cmd))
             return False, ""
-        except Exception as e:
-            logger.error(f"kubectl command failed: {e}")
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)